database/insertData/03_template_F_companyJoin.js

```diff
@@ -0,0 +1,319 @@
+                  }
+                  organisation_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.S1Q1.selection.id"]
+                  }
+                }
+              }
+              {
+                actionCode: "grantPermissions"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 130
+                condition: {
+                  operator: "AND"
+                  children: [
+                    {
+                      operator: "="
+                      children: [
+                        {
+                          operator: "objectProperties"
+                          children: [
+                            "applicationData.reviewData.latestDecision.decision"
+                          ]
+                        }
+                        "CONFORM"
+                      ]
+                    }
+                    {
+                      operator: "objectProperties"
+                      children: ["applicationData.reviewData.isLastLevel"]
+                    }
+                  ]
+                }
+                parameterQueries: {
+                  username: {
+                    operator: "objectProperties"
+                    children: ["applicationData.username"]
+                  }
+                  orgName: {
+                    operator: "objectProperties"
+                    children: ["applicationData.responses.S1Q1.selection.name"]
+                  }
+                  permissionNames: ["canApplyDrugRego"]
+                }
+              }
+              {
+                actionCode: "changeStatus"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 110
+                condition: {
+                  operator: "AND"
+                  children: [
+                    {
+                      operator: "="
+                      children: [
+                        {
+                          operator: "objectProperties"
+                          children: [
+                            "applicationData.reviewData.latestDecision.decision"
+                          ]
+                        }
+                        "CONFORM"
+                      ]
+                    }
+                    {
+                      operator: "objectProperties"
+                      children: ["applicationData.reviewData.isLastLevel"]
+                    }
+                  ]
+                }
+                parameterQueries: {
+                  applicationId: {
+                    operator: "objectProperties"
+                    children: ["applicationData.applicationId"]
+                  }
+                  newStatus: "Completed"
+                }
+              }
+            ]
+          }
+          templatePermissionsUsingId: {
+            create: [
+              # applyJoinCompany
+              { permissionNameId: 8000 }
```

Comment:
```suggestion
              # applyJoinCompany
              { permissionNameId: 8500 }
```

Or - can remove the new PermissionName: 8500 and use the applyGeneral:
```
// applyGeneral (associated with multiple templates)
  `mutation assignGeneralPermission {
    createPermissionName(
      input: {
        permissionName: { 
          id: 10100
          name: "applyGeneral",  permissionPolicyId: 2000 }
      }
    ) {
      permissionName {
        name
      }
    }
  }`,
```

Reply:
Also remember to add one action on the UserRegistrationTemplate to grant this permission to new user registered.